### ISSUE TITLE
Strategy: Where should Open Collective focus as a platform?

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#2168]
I want to start a strategy discussion that the Open Collective team and the wider community can contribute to. We've had some team workshops this week where these issues have come up, and we need to clearly lay them out and work toward some decisions.

## Context

Open Collective currently is not breaking even from revenue, although we have been trending in a positive direction. We need to figure out the right business model to become sustainable. We also need to be clearer about our strategy so the team and community can align with it. (note: you can find all our financial data in our public drive https://drive.opencollective.com)

## Potential Directions

The way I see it, these are the best strategic options we have:

### 1. Double down on Open Source

Host fees from the Open Source Collective are by far the biggest source of revenue. Open Source is where we have had the most traction. There is a clear gap that we fill enabling Open Source projects to raise and spend funds, especially connecting unincorporated projects with sponsors with big budgets.

We could focus everything on optimizing the platform for Open Source projects, onboarding more of them, and doing outreach and building our brand in the ecosystem. With the launch of things like Community Bridge and GitHub Sponsors, it's clear that this is a growth area, and that Open Collective has a unique offering (fiscal sponsorship + transparency + funding projects not individuals + being open source ourselves).

### 2. Focus on the next big community of Collectives

We could say that Open Source is already pretty successful and we should move on to focus on the next big area of traction. The most obvious one is Meetup groups. We could optimize features for networks of Meetups (like [Women Who Code](https://opencollective.com/wwcode), one of the largest most active hosts). This would likely entail improving our events functionality and/or integrating with external events platforms or meetup.com, plus doing a lot of marketing and outreach to this sector. 

Or potentially picking another big sector we have seen traction in, which also has money to work with. Another one that has shown early promise is grant-making organizations, who need a way to distribute and track the funds they give out, and want to fund unincorporated projects, which is hard for them without Open Collective.

### 3. Make the company about being a platform for fiscal hosting

Fiscal hosting is *the* feature that makes Open Collective unique, and solves serious pain points of administrating a fiscal sponsor organization. However, fiscal hosting is also the hardest thing to explain, and seems really boring unless you have a lot of experience with the specific pain points.

This would be more like a B2B approach, where we are a tool for fiscal host organizations, and they take care of the outreach to their own communities and sign on relevant Collectives. We'd have to follow more of a whale (vs minnow) approach, meaning building a sales team and longer lead-times on landing and onboarding customers, but bigger results when one does sign on (because each host brings many Collectives).

We could also improve the experience of onboarding as a self-hosted Collective (meaning an individual who uses their own accounts to hold funds for just their own Collective), which could enable a large number of small hosts to sign up. Right now that process is needlessly complex and we don't properly communicate about it.

## Pricing model

If we go with 2 or 3 above, we need to figure out the right pricing model for charging fees on funds coming in via other channels than credit card contributions, e.g. bank transfers, or funds the fiscal host already holds. Charging 5% on a $100,000 grant doesn't make much sense, but neither does charging $0. (See #1458)

Currently, we don't charge for funds fiscal hosts add manually, despite these being the largest chunks of money going through the system. For example, Women Who Code doesn't pay us much in fees despite being one of the largest and most active hosts. 

## Thinking strategically

We need to find the right mix of:

* What's already showing traction and we have evidence there is enthusiasm for from customers
* Alignment with our social values as a company and enabling positive impact on the world
* Leveraging what makes Open Collective unique and the special value we can provide that no one else is
* What's realistic given our runway and small team
* Aiming for customers that have money, both because our tool is for managing money and because we need a revenue stream

## Thoughts? Other ideas?

@piamancini @xdamman @cuiki @Memo-Es @lindatalu @Betree @znarf @benjam @raulgrafico @pietgeursen

**Comment by xdamman** [comment]
Thank you @alanna for putting all those thoughts together.

First, I'd say that the "Open Source Vertical" should make Open Collective sustainable on its own. Tech is the largest industry in the world. That's where a lot of money is. If we cannot be sustainable in that industry, we have a larger issue.

To reach sustainability, I'd explore offering services with more added value that would justify taking more than 5%. The first one that comes to mind is the subscription model to make it dead easy for companies to "back your entire stack". See https://github.com/opencollective/backyourstack/issues/168
I think it's in the interest of the community to mandate Open Collective to do the work and reach out to tech companies to get them on such a subscription plan and then trickle down the money based on their dependencies. Asking 30% instead of 5% seems justified in this case because it's much more than just offering accounting (and accounting software), it's reaching to out to sponsors, managing purchase orders, and developing (and iterating) on an algorithm with the community to spread that money in a fair way (as a reference, a classic ad agency takes around 50%, which publishers are happy to do as it enables them to not have to ever deal with advertisers, they can focus exclusively on their work).

Second, I think that it's our morale duty as a company to join the movement to declare the state of climate emergency. As such, this is not business as usual. It cannot be business as usual.

One of the biggest movements attacking this problem is [Extinction Rebellion](https://rebellion.earth/).
Like with Women Who Code, we can help. We should help.

There are already a couple of local chapters of the movement on Open Collective. I'm myself contributing to [Extinction Rebellion Belgium](https://opencollective.com/xr-belgium).

I think that as the open source community, we can and we should help them. It's a joint effort. And this is how we can contribute.

Last but not least, I think it's always good to go back to the roots. In [one of the initial blog posts to explain the vision](https://medium.com/open-collective/a-new-form-of-association-for-the-internet-generation-part-2-fe6d8415f444), I wrote:

> We need to build software to enable any organization to easily host a new ligthweight form of association for the Internet generation: open collectives.

> If we can do this, we will foster an explosion of bottom up initiatives that will not be limited to activities that don’t require money. Those initiatives will finally be able to collect money, have a budget and have a larger impact.

I wouldn't see this as B2B software. That sounds boring. I'd see this as a federation of people who want to enable a bottom up revolution wherever they are. Wether it's in Paris with [Open Collective Paris](https://opencollective.com/paris), Brussels with [Open Collective Brussels](https://opencollective.com/brussels), London and the UK with [Open Collective UK](https://opencollective.com/uk) in Open Source with [The Open Source Collective](https://opencollective.com/opensource), in Open Source in Africa with [Open Source Community Africa](https://opencollective.com/osca), etc.

The best analogy I have for this is [Wordpress](https://en.wikipedia.org/wiki/WordPress). It was a big catalyst in the blogging revolution. It removed a lot of the friction to start your own blog. Thanks to this open source software we went from a few thousands publishers around the world (mostly state or corporation owned) to millions.

Today, the world needs the same revolution to happen but not in the realm of bits, in the realm of atoms. Taking initiatives in meat space should be simpler. States and corporations should not have the monopole of taking initiatives to shape the world we live in.

*TLDR: let's make open collective sustainable in the tech industry where the money is, and let's give it to the world to accelerate the transition from corporations to communities.*

Note: The fact that we do everything in open source could already be considered as giving it to the world (though arguably we should invest more in making it easier for people to install the software on their own server).

Now, if we are hosting the software for the fiscal host, then we should charge something (for maintenance, support, etc.). We talked some more on Wednesday evening about the problem of not charging anything for money that comes in via bank transfers. We thought that we could charge a monthly fee that scales with the total amount of funds managed by the fiscal host. That way, we don't charge directly the donor. We don't take 5% before the money gets into a collective. But we bill the host every month. The host becomes this way the only one taking a commission when the money comes in (+ [Stripe](https://stripe.com) for credit card payments). This will simplify things a lot for the end user. And this will turn Open Collective into a federation of hosts where our focus will be to empower hosts to be as successful as possible. That will scale better.
The alternative is to keep a direct relationship with every collective but then we would fall back in the trap of existing tech platforms that are way too much centralized. That's not where the world is going.

**Comment by alanna** [comment]
OK so to summarize what you said @xdamman - double down on open source (esp the Back Your Entire Stack subscription model), and use success there to resource helping other kinds of groups in the future that don't have such a clear business model. I think that makes sense. I'm not 100% sure it's better than the other possibilities discussed here (we would need to validate BYES subscription in the market first), but I could see how it could work. 

*However* to get there we would need to absolutely focus on BYES and the open source vertical, and put down all the other distractions and auxiliary strategies and projects. As a tiny team we cannot afford split attention if we're going to nail any of the potential directions we might choose. Whatever conclusion comes out of this discussion, we need to narrow down our focus and really go for it.

**Comment by xdamman** [comment]
Not exactly, I’d argue that we should be a platform for collectives to receive contributions to make them sustainable. Starting with the contribution of offering a service of fiscal sponsorship. The Open Source Collective (OSC) is one of them. 

In the ideal world, the OSC would take on the BYES project because it is clearly 100% open source community focused. It would also make things easier for everyone to understand why the fees are different than the classic OC fees. It would basically be a new service of the OSC that people can opt in to to be part of a bundle that the OSC can start “selling” to companies who want to have an automated subscription service. After all, it’s anyway the OSC that will have to do the work of liaising with companies, becoming a preferred vendor, handling their purchase order process, answering questions, etc. That’s work and that solves one of the top requests of the community (according to the survey) which is helping them finding sponsors. That’s worth a lot.

That’s how it works in the Drupal / Wordpress world. There are plenty of organizations / companies that finance the development of special features for their own needs. Then, they share those new features with the larger community.

The platform’s responsibility is to develop the features that are common to all users and make it easy for anyone to adapt the platform to their local needs.

What I like about that is that it will help focus the company.

**Comment by piamancini** [comment]
Before I go deep into the conversation (it's tough having to re-argue things again, need some strength!) :) I am starting a company in Spain to invoice Open Collective Inc. I have a ton of receipts and small expenses to keep track of.  My plan is to start a new collective, with one host to add funds and manage the expense flow of the company - it helps a lot with tax revenue. 
I'd gladly pay $10-$20 a month for this. Adding funds should be a paid feature.

**Comment by timschloe** [comment]
Ehm, this... is a very exciting debate at a very exciting time... Thanks for tweeting Pia!

Would it be possible to interview some one here on Transformational Change / Paradigm Shift from a Climate Finance perspective? (Research in that field is my dayjob and I see a giant overlap to what you are doing here)...

**Comment by BenJam** [comment]
> it's tough having to re-argue things again

💯

> we could charge a monthly fee that scales with the total amount of funds managed by the fiscal host

IMO this is a much friendlier way of saying:

> Adding funds should be a paid feature

In general I'd vote for keeping the opencollective.com vertical-free, providing a fair share of money managed from *all* hosts to OC Inc. and being more forward about the deal for hosts: "%age/fee to platform, %age to you and X projects currently looking for a host". This means by default OC supports 'hosts of one project' and may need to make improvements to help projects find a host. 

For me this doesn't turn open collective into a host sales team. It provides a space for everyone to start a project and manage finances on behalf of it openly, a pathway to remove personal legal and financial risk and a realistic 'money on the table' deal for anyone willing and able to take on those risks.

But whatever Open Collective choose to do, you should make sure that you view it from the outside in. Ask yourself how your users see themselves today, what they want to become tomorrow and how they will see you as the pathway to do help them achieve that 🤙

**Comment by alanna** [comment]
@piamancini which part do you feel you are having to argue again? I understand that these conversations have been happening for a long time, but even as a team member I do not know what the conclusion has been. I think this speaks to the need we identified in the recent workshop: get better at clarifying the outcomes of decisions so we don't need to revisit them repeatedly. 

@BenJam what you are saying sounds like vote for option 3 to me, but you are pointing out that the nature of the funnel could be different to what I described originally. I am the one providing a lot of the host onboarding support currently, and I said it would be more high-touch sales because of the level of support hosts need to understand our offering and get set up. However, that process could be *greatly* improved by fixing the host onboarding flow, creating a host guide (docs), and marketing specifically to hosts in a way they understand. You might be right that we could make progress with a low-touch approach.

@timschloe - @xdamman has been focused on working with climate action groups specifically so he may be the best person to interview.

My fear in this discussion is we will end up back where we are now, trying to sort of do all strategies at once and keeping all the doors open. Strategy is about what you are saying *no* to as much as what you are saying *yes* to, because saying yes to everything when capacity is limited is as good as saying yes to nothing.

So another way to approach this discussion would be: **What will we say *no* to in order to enable a stronger *yes*?**

**Comment by salaboy** [comment]
@alanna @piamancini 
Some outsider comments here, about the **Potential Directions** without having enough insights of the history of Open Collective, but with having 12 years of Open Source background (ex Red Hat/JBoss, ex Alfresco and contributor of a number of Open Source projects) 
I found the idea of Open Collective amazing, and I would love to see this all over the place to raise awareness of how Open Source can be a viable way of make a living without being tied to a company. 

So totally **Double down on Open Source**, but I would like to add some suggestion on where to focus:
- The CNCF foundation (https://www.cncf.io) should have this, and I see as a big investment for Open Collective to engage with such foundation to make other smaller projects aware on how Open Collective works
- I've heard about "back your stack" (https://medium.com/open-collective/introducing-backyourstack-39747ccdf2a5) and once again, this should become a must for serious players, again foundations like CNCF should have this 
- Maybe target other foundations such as the Apache Foundation (http://apache.org), which are big hubs of mature open source projects which have more problems to keep the communities thriving. 

I do see things like CNCF, as a must to validate that the platform and the business model works. Having said this, I would strongly consider giving the services for free to them, because today, if you are in the Open Source space, you are part of the CNCF.  

Regarding **Focus on the next big community of Collectives**, I do believe that sponsoring meet-ups in the same way as bounties are sponsored sounds like a great idea to me. Having some kind of partnership with co-working spaces where you can use part of your credit instead of sending an invoice might be another good idea. 

Regarding the **Pricing model**, even if I don't fully understand it right now, 5% of the funds makes a lot of noise. My first though was more based on number of bounties closed, meet-ups sponsored, companies joining as sponsors, so it has a direct correlation to what is going on in the community, if the community is active Open Collective is successfully helping and hence it gets more money. This can also be using a subscription model, meaning, once a month based on the activity. 

So in order to make sense to developers and maintainers it needs to: 
- Don't consume the project funding, unless they can see the results (after the results, not before)
- Help the project to get more funding, offering reports of which companies are sponsoring the project and with how much
- Give companies the right data, so they can choose which projects to sponsor (back your stack is a great idea for this) 
- Open Collective should be a very active organisation in Open Source related conferences

Hopefully these make some sense.. 
I am happy to have more in detail conversations, but I didn't want to keep adding stuff in this already long issue :) 

Cheers

**Comment by BenJam** [comment]
> sounds like vote for option 3 to me

pretty much. 

> that process could be greatly improved by fixing the host onboarding flow

++ I see flows for registering a new 'host-less' project, advertising a project as needing a host/finding existing hosts and for hosts to find projects.

**Comment by alanna** [comment]
@xdamman and I had a good conversation about strategy stuff, and I want to capture notes here while it's still fresh. After going over everything, this is what emerged:

## A. The Open Source Collective should double down on open source
To do that, it needs to become a much stronger independent organization. We've taken small steps in that direction this year (hiring me as exec director, working on a new brand, kickstarting board meetings, taking on OSC-specific initiatives, and paying OC inc for OSC-specific features out of its own budget). But we have to go further.

The first big next step should be to hire someone who works just on OSC. The budget is there. This person can focus on bringing on more Collectives and Sponsors, designing and building up premium services that open source projects need and find worth paying for, building a bigger presence in the OSS ecosystem. OSC should pay for its own developer capacity to build OC platform features for OSS Collectives.

OSC should be the entity building things like Back Your Entire Stack and the other initiatives focused on Open Source. There is *huge* potential, resourcing, and opportunity in the open source sector, and we already have strong traction. But to realise that potential, OSC needs a focused skilled team.

## B. Change the business model
OC inc charges fiscal hosts $10 per month per (active?) Collective. Fiscal hosts charge their Collectives however they want, without that being muddled with the OC fees. OC inc focuses on building a platform for many ecosystems, OSC being one, and onboarding and growing fiscal hosts and their Collectives.

Charging a percentage on donations is a model that works for some fiscal hosts and not for others. Here's how the new model could potentially play out in different contexts:

* OSC pays $12,000 a month to OC inc for its 1200 Collectives (approx), which is pretty much the same it's paying now through the 5% platform fee. OSC charges its Collectives 10%. Functionally, this is not a price change for Collectives (who are currently paying 5%+5%). It also focuses on developing premium value-add services (like Back Your Entire Stack) which are more than worth it for Collectives even at higher fee levels because it solves serious pain points around finding sponsors, marketing, etc.

* WWcode (or another host like them, if they are grandfathered into their current deal) pays $840/month to OC inc for their 84 Collectives. (They pay Meetup.com $30/month per group, three times as much). This allows OC inc to get a reasonable fee for the value they get, while not charging crazy percentages on the large amounts of money they credit to Collectives through their host org. WWcode doesn't charge their Collectives anything.

* Self-hosted Collectives pay $10/month. 

* OC Europe, OC UK, OC Paris , and all other hosts decide for themselves if they want to charge their Collectives for their services and to help cover the $10/month/Collective fee to OC inc, and if they want to charge as a % on income or a set monthly fee.

## C. Open Collective Inc pursues strategy 3 above, becoming a platform for fiscal hosts
This will probably involve some of strategy 2, focusing on specific ecosystems who are likely to come on board with their communities of Collectives, but only insofar as it will improve the platform for everyone. This is aligned with a business model where OC inc's paying customers are the hosts.

## D. Xavier branches off to focus on XR/Agora
For the last year, Xavier has been exploring the climate action ecosystem and is convinced that if we can get some of these groups on OC now, we can scale with them as they are replicated around Europe and the world. Just like OSC should be its own seperate org focused on its ecosystem, this area needs its own org and identity, and Xavier probably needs to raise money and build a team if he really wants to go for it.

We can't sustain the ambiguous situation that we've had for the last year where Xavier has one foot in that world and one foot in the OC team, becoming a distraction. It's not good for him and it's not good for the rest of the team. In the context of OC inc as a platform for many different communities, as the XR/Agora area grows and gets traction, resources, and clarity, that team can contribute to the codebase and improve the platform for all, or provide resources and justification for specific features it wants to build (like OSC or any other host). 

## Next step 
We agreed that a three way conversation with me, @xdamman and @piamancini to discuss all this further would be a good idea. Our perception is that the team would be much happier with strategic clarity, and while they have important perspectives to add, they are kind of waiting for the cofounders. My hope is I can help facilitate progress and clarity with them. That said very keen to hear from @Betree @znarf @cuiki and any other team members or community voices.

**Comment by piamancini** [comment]
Thanks everyone for the comments! 

## Vote
My vote/proposal is to go towards a vertical agnostic platform. This means that we focus our work on enabling any community to raise funds, manage their funds and expenses transparently supporting themselves on an existing fiscal host. 

When we started Open Collective we set out to create a minimal organizational structure for communities to operate in this world, removing from them the complexity of having to deal with accounting, lawyers, etc. We want to unlock the potential of communities blocked by an existing system that was built in a different era. I still believe in that. 

There's two ways of achieving this, we either become ourselves the fiscal host of a community (like we did for the open source space) and build for that community or we enable anyone to become a fiscal host. I think that doing the latter, also serves the former. By building the best platform for fiscal hosts to support their networks we will be also serving the open source community via the Open Source Collective, Wordpress Foundation, Dot Net Foundation (all fiscal hosts on the platform today) to better grow and support their ecosystems. 

For example: reaching out organizations like the ones mentioned by @salaboy will be part of our outreach and we'll be building tooling that helps them, since they are essentially Fiscal Hosts. 

Being vertical agnostic is also a forcing function to focus: we need to build for all, not for one. So new features need to be useful for everyone or at least large subsets of our collectives. I think this will be positive for the development of Open Collective as we've been suffering from ad hoc and quick fixes for specific users. 

Better expense management, a strong well-built ledger that handles multiple currencies, better reporting for hosts, governance systems in place, multiple payout methods, better distribution of funds among collectives... These are features that are valuable for everyone. I am slightly concerned about @alanna's comment above: 

> This will probably involve some of strategy 2, focusing on specific ecosystems who are likely to come on board with their communities of Collectives, but only insofar as it will improve the platform for everyone.

We have a history of starting and dropping features for potential ecosystems. Going forward we should be a lot more rigorous in our approach to this. 

> For me this doesn't turn open collective into a host sales team. It provides a space for everyone to start a project and manage finances on behalf of it openly, a pathway to remove personal legal and financial risk and a realistic 'money on the table' deal for anyone willing and able to take on those risks

I agree with this and it summarizes well my thinking. 

## Caveat
One caveat: We are currently developing a way for companies to support their dependencies on Open Collective. A Back Your Stack V2. While this qualifies as building something for a subset of our collectives, the open source community, I think we should finish it as it supports the largest community on the platform and one that we have been actively growing for the past years. While the Open Collective Inc. team will build the basic functionality of BYS v2, it is the Open Source Collective Fiscal Host who has to decide how and if they reach out to companies for this. 

@alanna's comments on https://github.com/opencollective/opencollective/issues/2168#issuecomment-508432808 point A are clear. There's a place for the Open Source Collective to scale up their operations - I am personally supporting this effort from the board and will continue to. 

This feature is also strategic if we want to collaborate more with GitHub becoming the way for projects to receive funds as a community. Growing from the funding.yml to a more evident path. 

## Business Model

There's still an ongoing conversation about this we are having with @xdamman. We should be able to post some numbers soon. The gist of it is to charge Fiscal Hosts for a $10-$15 /m subscription for enabling them to add funds manually to their collective + 5% of funds raised through the Open Collective Platform. We'll start a new thread with this, but current questions are, what we do with inactive collectives? How and should we grandfather in existing hosts? Should there be a minimum free version?

**Comment by coderberry** [comment]
Boy am I sad I'm late to this thread. I am a HUGE fan of Open Collective and the team behind it. I believe you have created something that provides incredible value to the community.

I would like to suggest that a strong partnership (or more?) with CodeFund would really open up the ability to generate profit. CodeFund provides a solid path to profit, but doesn't have the network nor influence of Open Collective.

If this is interesting at all, let's talk and talk soon. I think we can help!

**Comment by alanna** [comment]
Thanks so much @coderberry ! I would love to talk more. Let's have a call. I will reach out to you directly to set that up.

**Comment by alanna** [comment]
Myself @piamancini and @xdamman had a conversation about everything in this thread, and we feel it's a good time to close this discussion. Strategy is always an ongoing conversation, but we think we've reached a good level of clarity for the medium term future.

Pia's last comment is the basic conclusion, but I'll summarise it here for clarity.

* Open Collective will aim to be a platform for all kinds of diverse fiscal hosts, with features that serve all of them.
* Our goal will be to bring on more fiscal hosts and help them thrive, and those hosts will bring their communities and networks of Collectives.
* Open Source Collective will continue to differentiate itself from Open Collective Inc, as the home of specifically open-source Collectives. There will be a transition process over time so this won't be overnight.
* Open Collective Inc will continue supporting work for the open source community, such as the expansion of Back Your Stack already in the works, because this also will help all hosts (by enabling people to contribute to a host who will then distribute funds to Collectives, which other hosts might want to do). However in the future this sort of thing will be more firmly under OSC.
* We envision a future where a council of active hosts is involved in platform governance, and they, as thriving communities, may build or co-fund features they want.

Actions from here:

* @piamancini will ensure the core team is up to speed with all of the above and that they can have input. The team will figure out how to align its work with the strategy.
* @xdamman is leading out on evolving the business model, [being discussed in this thread](https://github.com/opencollective/opencollective/issues/2251).
* Xavier will clarify the shifting focus of his role and communicate how it will align with the above once he's had some time to work it out.

**Comment by BenJam** [comment]
Is there a living document anywhere for contributors/supporters?
